module-3/web-app/src/components/Pages/Projects/Projects.jsx

```diff
@@ -0,0 +1,25 @@
+import React, {useState, useEffect} from "react"
+import  axios from 'axios'
+import style from "./Projects.module.scss"
+
+const Projects = () => {
+    const [projects, setProjects] = useState('')
+   
+    useEffect(() => {
+        axios.get("http://localhost:3001/selectAllProjects")
```

Comment:
I suppose it isn't the final version, can you prepend this block with //TODO comment to make is explicitly clear that it's in dev mode still

Reply:
Instead of axios use native fetch api as we did for prosemirror in this draft pr https://github.com/USergei/bootcamp/blob/9b9cf016678c528a2635be3c7d8053f0b9720024/module-3/web-app/src/services/documentService.js

It's better to have as little external dependencies as possible

Reply:
Done